### DISCUSSION TITLE
No mouse cursor

### DISCUSSION THREAD

**Discussion by ElliottBridge** [rern/rAudio#835] [Fixed Issues]
Trying to use it on my Pi4, is incredibly difficult, occasionally I can get buttons to highlight, but can't select certain buttons with my mouse since there is no cursor present.

**Comment by rern** [comment]
Features > Browser on RPi > settings

**Reply by ElliottDyson** [reply]
Thank you, I was eventually able to get the network preconfigured so I was able to access the settings on my phone and enable it, however it might be useful to have it initially enabled upon first launch in case of issues like mine, as from the menu it is much easier to add a network rather than having to preconfigure one 👍
